test/clj/rems/api/test_catalogue_items.clj

```diff
@@ -58,6 +58,13 @@
                 (is (response-is-not-found? response))
                 (is (= "application/json" (get-in response [:headers "Content-Type"])))))))
 
+        (testing "create with mismatched organization"
+          (let [data (create-catalogue-item "owner" "nbn")]
+            (is (not (:success data)))
+            (is (= [{:type "t.administration.errors/organization-mismatch"
+                     :form form-id :resource res-id :workflow wf-id}]
```

Comment:
Should it show the organizations here?

Reply:
hmm, why not. any suggestions for the structure of the error? two ideas:
```
{:form {:id form-id :organization "foo"} :resource {:id res-id :organization "boing"}}
{:form-organization "foo" :resource-organization "boing"}
```

Reply:
I think the first one makes more sense.